code/datums/spells/touch_attacks.dm

```diff
@@ -56,6 +58,15 @@
 
 	action_icon_state = "gib"
 
+/obj/effect/proc_holder/spell/targeted/touch/disintegrate/cast(list/targets)
+	for(var/mob/living/carbon/user in targets)
+		visible_message("<span class='danger'>[user] starts gathering destructive energy...</span>", "<span class='danger'>You start gathering destructive energy...</span>")
+		playsound(user.loc, 'sound/magic/Ethereal_Exit.ogg', 50, 1)
+		if(!do_after(user, 15))
+			return 0
+		visible_message("<span class='userdanger'>[user] channels a large amount of destructive energy to his hand!</span>", "<span class='danger'>You channel a large amount of destructive energy to his hand!</span>")
```

Comment:
"You channel a large amount of destructive energy to his hand!" Something seems off there